### ISSUE TITLE
508: additional usability improvements

### ISSUE THREAD

**Issue by leahbannon** [18F/fbopen#107]
- List of links should be in <ul> tags. This provides someone using a screenreader with a prompt indicating how many links are about to be read to them. ex. the social media links. 
- Hidden header tags or aria role elements should be used to denote the different sections of the site to make navigation easier. 
- The enter key is used to submit the search field, this can be confusing when trying to select advanced menu options.
